templates/cohorts/new_cohort.html

```diff
@@ -195,6 +195,24 @@ <h4 class="panel-title pull-left">Selected Filters</h4>
             </div>
         </div>
 
+        <div class="cohort-info panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">Details</h4>
+            </div>
+            <div class="panel-body">
```

Comment:
It's being defined in the CSS classes, but I can have a look at bootstrap maybe. Do we have any kind of documentation for it?